src/Ast/Visitor/NamespaceSubstituteVisitor.php

```diff
@@ -36,9 +36,9 @@ public function leaveNode(Node $node)
      */
     private function renameNode($namespacedStatement): void
     {
-        if($namespacedStatement->name !== null) {
+        if ($namespacedStatement->name !== null) {
```

Comment:
I am preparing for another PR, but I miss phpstorm configs. Could you import and share the config, so we would have same code style?

Reply:
hmm I use vim 😄 but for this PR all I've done was running php-cs-fixer which uses psr1/psr2 standards